### PR TITLE
Created basic interface

### PR DESCRIPTION
**Pull request by ameerhamzakhalid**
-     Adds app file
-     Adds main file
-     Fills the UI accordingly

### CODE REVIEW

File: main [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  input = nil
+  until input == 7
+    display_options
+    input = gets.to_i
+    puts 'Invalid choice, please select a number from [1..7]' if input < 1 || input > 7
+    input_process(input, app)
+  end
+end
+
+main
```
Comment (DeliceLydia) [inline main]: - [ ] It is main.rb you forgot `.rb`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp.capitalize
+    print 'Specialization: '
+    specialization = gets.chomp.capitalize
+    teacher = Teacher.new(age, name, specialization, person_type: 'teacher')
+    @people << teacher
+    puts ['Person created successfully', ' ']
+  end
```
Comment (DeliceLydia) [inline app.rb]: - [ ] I am not able to create a `student` or a `teacher`
Screen Shot 2023-03-10 at 12 22 17

Comment (DeliceLydia) [reply app.rb]: - [ ] Remember to remove all the examples used in previous project

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ameerhamzakhalid, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Clean codes and organized
- Followed git workflow

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @ameerhamzakhalid, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Your app works fine 👍 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**